examples/user_remap_script.py

```diff
@@ -38,4 +40,4 @@ def remap(files_tree: dict):
 
         chunks.append(chunk_files)
 
-    return chunks
+    return chunks, classes
```

Comment:
```suggestion
    return chunks
```
you changed API interface